InvenTree/InvenTree/helpers.py

```diff
@@ -151,6 +151,7 @@ def download_image_from_url(remote_url, timeout=2.5):
             timeout=timeout,
             allow_redirects=True,
             stream=True,
+            headers={"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64)"},
```

Comment:
I am not sure it is in good faith to immitate a user agent. I would rather not do that by default.

Reply:
I can see your point, the change has been reverted.

Reply:
What about adding this as an option in the settings?

Reply:
Added a setting to allow the default user-agent to be overridden.